qt/python/mantidqt/widgets/fitpropertybrowser/addfunctiondialog/test/test_addfunctiondialogpresenter.py

```diff
@@ -0,0 +1,45 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Thanks for adding the test. I think it's missing from the test list in [`CMakeLists.txt`](https://github.com/mantidproject/mantid/blob/master/qt/python/CMakeLists.txt) so `ctest` will skip it.